.github/workflows/release.yml

```diff
@@ -7,152 +7,77 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
```

Comment:
Hi @carmal891,
Should we consider retaining this as `ubuntu-latest`? I had read that the warning was just a kind of notice to keep users informed that latest would be pointing to ubuntu-24.04, and this rollout would be complete by 17th January.

This can help us avoid bumping the runner image in a different PR at a different point in time.

Ref: https://github.com/actions/runner-images/issues/10636